### PR TITLE
solved issue #436

### PR DESCRIPTION
**Pull request by Krish-bhardwaj**
# Description
changed the letter spacing to 0 solved issue #434

### CODE REVIEW

**Comment by Krish-bhardwaj** [comment]
hi @QuaidBartolomei  i have made the suggested changed plz look after my PR

**Comment by dadiorchen** [comment]
@Krish-bhardwaj no need to close and create PR next time, just directly push up your branch works

**Comment by QuaidBartolomei** [comment]
@Krish-bhardwaj I'm still encountering the same issue with these changes.

2022-01-22 01 12 54 localhost 81644b97438f

**Comment by dadiorchen** [comment]
@QuaidBartolomei yea, that's what surprised me, and I can not find the source of this settings

**Comment by Krish-bhardwaj** [comment]
Hi i have made some change @dadiorchen and @QuaidBartolomei  ig this solves our issue !

File: src/theme.js [hunk 1]
```diff
@@ -39,11 +39,24 @@ const colorTheme = createTheme({
...
+      styleOverrides: {
+        // Name of the slot
+        root: {
+          // Some CSS
+          letterSpacing: 0,
+        },
+      },
+    },
+  },
+});
```
Comment (dadiorchen) [inline src/theme.js:55]: is it possible to declare two const with the same name?

**Review by VWRoli — CHANGES_REQUESTED** [review]
https://github.com/Greenstand/treetracker-web-map-client/blob/3eeb17cdd55123a593c932ed531f36de7e8db2e6/src/theme.js#L42-L55

We can't create a new theme because we already have one. And also you try to change letterspacing on the button component, and the problem is the h1 component.

https://github.com/Greenstand/treetracker-web-map-client/blob/3eeb17cdd55123a593c932ed531f36de7e8db2e6/src/theme.js#L111
This also has nothing to do with the issue as this is for the body1 component, which should have a default letter spacing.

https://github.com/Greenstand/treetracker-web-map-client/blob/3eeb17cdd55123a593c932ed531f36de7e8db2e6/src/theme.js#L62-L70
H1 element probably has a default letterspacing defined by MUI, so we can't remove it, just override it.
If we put ` letterSpacing: 0` into this part of the code, where we define the default for `h1` component we override the default negative letter spacing.

Screenshot from 2022-01-29 14-37-45

File: src/theme.js (hunk 1 above)
Comment (VWRoli) [reply src/theme.js:55]: No, it is not.

**Pull request closed without merging**